sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Release History
 
+## 12.3.0 (2020-01-15)
```

Comment:
Not sure we actually need to release this?

Reply:
It has changes for one of the blob things